### DISCUSSION TITLE
[4.0] * not explained on installation

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#47132] [2. Cold Features]
### Steps to reproduce the issue
open 1st installation screen.
look at label `Enter the name of your Joomla site. *`

### Expected result
Explanation for the *

### Actual result
No information provided what it means.

### System information (as much as possible)
4.0-rc4

### Additional comments
Remove the * because every field is required on this site or add an explanation.

**Comment by dgrammatiko** [comment]
Just add a label layout override in the installation template that doesn't have the stars, an easy fix

**Comment by brianteeman** [comment]
These are observations - not saying they are good practice

1. Nowhere in the CMS do we do anything other than use a * as a visual indicator that a field is required so its not just the installation
2. On a form where the majority of the fields are required I would sometimes just mark the non required fields as optional. But that doesnt work for a complete cms as we have consistency issues across forms

Additionally this [advice/observation](https://www.nngroup.com/articles/required-fields/)  from the Neilson Norman group observes that the use of a * is commonplace across the internet and so following Jakob's Law of Internet Experience we shouldn't need to change it.
